src/ElmSharp/ElmSharp/Size.cs

```diff
@@ -88,6 +93,11 @@ public override bool Equals(object obj)
             return Equals((Size)obj);
         }
 
+        public string Test()
+        {
+            return string.Format("{{Width={0} Height={1}}}", Width, Height);
```

Comment:
The behavior of 'string.Format(string, object, object)' could vary based on the current user's locale settings. Replace this call in 'Size.Test()' with a call to 'string.Format(IFormatProvider, string, params object[])'.